### ISSUE TITLE
I have another piece of CSS for a more fluent hover to display effect

### ISSUE THREAD

**Issue by sethyuan** [YU000jp/logseq-plugin-left-sidebar-enhance#1]
Hi @YU000jp , great plugin, I wonder if you can integrate the below CSS into your code, it has a more fluent hover to display effect:

```css
#left-sidebar {
  transform-origin: center left;
  scale: 0.05 1;
  opacity: 0;
  transition-property: opacity, scale;
  transition-duration: 80ms;
  transition-timing-function: ease-in-out;
}
#left-sidebar:hover {
  scale: 1;
  opacity: 1;
}
#main-container.is-left-sidebar-open {
  padding-left: 0;
}
```

**Comment by YU000jp** [comment]
If a shorter CSS code is sufficient, I will choose your code.
The original CSS code is created by mæn(https://discord.com/channels/725182569297215569/775936939638652948/1155251493486727338).

I am using your "favorite tree" plugin. I wanted to use these together. I found it inconvenient that the width adjustment feature was unavailable in custom CSS. Therefore, I decided to create this plugin to enable toggling of the button in the top-left corner.
